ersilia/core/model.py

```diff
@@ -421,9 +422,10 @@ def update_model_usage_time(self, model_id):
 
     def serve(self):
         self.close()
-        self.session.open(model_id=self.model_id)
+        self.session.open(model_id=self.model_id,track_status=self.track_status)
```

Comment:
Let's keep it as `track`, or `track_runs`.

Reply:
Done